.github/workflows/build-tag-publish.yml

```diff
@@ -74,3 +74,35 @@ jobs:
           JFROG_USERNAME: ${{ secrets.JFROG_USERNAME }}
           JFROG_PASSWORD: ${{ secrets.JFROG_PASSWORD }}
         run: ./gradlew publish -Pversion=${{ steps.get_tag.outputs.semVer }}
+
+      - name: Login to LinkedIn Jfrog Docker artifactory
+        if: ${{ success() && github.ref == 'refs/heads/main' }}
```

Comment:
could we also add a condition saying its not a fork and it linkedin/openhouse?

Reply:
sure, fixed